data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/AggregateProcessor.java

```diff
@@ -116,11 +119,15 @@ public Collection<Record<Event>> doExecute(Collection<Record<Event>> records) {
             final Event aggregateActionResponseEvent = handleEventResponse.getEvent();
 
             if (aggregateActionResponseEvent != null) {
+                aggregateActionResponseEvent.getMetadata().addTags(List.of(AGGREGATED_TAG));
```

Comment:
This may be a breaking behavior. Anybody who is currently outputting tags as a means to detect failures is now going to get tags that indicate normal events.

I think we should do one of the following instead: 1) Require the user to provide this tag; 2) Only include this when the user configured to send the unaggregated events; or 3) Tag the unaggregated events with an `unaggregated` tag.